src/datasource-zabbix/datasource.js

```diff
@@ -148,11 +147,11 @@ class ZabbixAPIDatasource {
     });
 
     // Data for panel (all targets)
-    return Promise.all(_.flatten(promises))
+    return emptyPromise.then(() => Promise.all(_.flatten(promises))
```

Comment:
Why do we have to wait for `alertQuery()` response?

Reply:
Look at the screenshot in description. Without that some alerts had wrong background color because thresholds are set while fetching alerts. 
At first screenshot the last alert has red heart but green background. It should be orange like on second screenshot.